packages/components/src/utils/observeProgressBarRect.js

```diff
@@ -106,8 +106,18 @@ function refresh() {
 }
 
 function updateScroll() {
-  scrollTop = document.body.scrollTop;
-  scrollLeft = document.body.scrollLeft;
+  const supportPageOffset = window.pageXOffset !== undefined;
+  const isCSS1Compat = (document.compatMode || '') === 'CSS1Compat';
```

Comment:
I think we can skip "isCSS1Compat" so get rid of all the stuff with document.documentElement.

Reply:
the problem is that document.documentElement is supported for Ie: https://developer.mozilla.org/en-US/docs/Web/API/Document/documentElement

so I think we might need it. 

this is another suggestion: 

  const supportPageOffset = window.pageXOffset !== undefined;
  const BackCompat = document.compatMode === 'BackCompat';

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Document/compatMode

Reply:
Ok i removed document.documentElement :-) I can confirm that everything is still working